### ISSUE TITLE
Work with Alchemer?

### ISSUE THREAD

**Issue by xhostcom** [ylecuyer/survey-gizmo-ruby#113]
Hi does this work now that its called Alchemer? Thanks

**Comment by ylecuyer** [comment]
Yes alchemer is supported, it is clearer with https://github.com/ylecuyer/survey-gizmo-ruby/pull/118 pointing to new alchemer urls. Maybe in the future we will need to change the gem name too
